src/mirage/mirage.ts

```diff
@@ -0,0 +1,47 @@
+import { createServer, Model } from "miragejs"
+import imageProduct1 from '@/assets/images/image-product-1.jpg'
```

Comment:
What do you think of using named exports for readability?

Reply:
Named exports ensure readability across multiple files such that developers are aware of what type of export they're working with based solely from the name of the export. I am willing to change the import names if you have any suggestions.